api/src/routers/upload.py

```
@@ -0,0 +1,200 @@
+from typing import Annotated
+from pathlib import Path
+import hashlib
+import uuid
+import time
+import aiofiles
+from fastapi import UploadFile, Depends, HTTPException, Form, APIRouter
+from fastapi.security import OAuth2PasswordBearer
+
+from shared.models import Metadata, MetadataPayloadData
```

Ruff (0.8.0)

`shared.models.Metadata` imported but unused

Remove unused import

(F401)